server/SCHEMAS/mongodb.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require("mongoose")
+const mongoDB = "mongodb://127.0.0.1/qa"
+
+mongoose.set('strictQuery', true)
+mongoose.connect(mongoDB, { useNewUrlParser: true, useUnifiedTopology: true })
+
+const db = mongoose.connection
+db.on("error", console.error.bind(console, "MongoDB connection error:"))
+
+// Define schemas
+const Schema = mongoose.Schema
+
+const AnswerSchema = new Schema({
+  asnwer_id: Number,
+  body: String,
+  date: Date,
+  answerer_name: String,
+  helpfulness: Number,
+  photos: Array,
```

Comment:
Not sure if this is an array of objects with key/value pairs, but you may want to define the datatypes inside this array as well. Looks great, love the screenshots!